tests/subsys/suit/cache/testcase.yaml

```diff
@@ -1,7 +1,7 @@
 tests:
   suit-platform.integration.suit_cache:
-    platform_allow: nrf52840dk/nrf52840 native_posix native_posix/native/64
+    platform_allow: nrf52840dk/nrf52840 native_sim
```

Comment:
note that there is also a native_sim/native/64
Which depending on wanting to test 64bit configuration may still make sense in the platform_allow

Reply:
done